querier: limit remote read parallelism (#11892)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

# CLI flag: -querier.filter-queryables-enabled
[filter_queryables_enabled: <boolean> | default = false]

> # (advanced) Maximum number of remote read queries that can execute
> # concurrently. 0 or negative values mean unlimited concurrency.

Reviewer: ```suggestion
# (advanced) Maximum number of remote read queries that can run
# concurrently. 0 and negative values indicate unlimited concurrency.
```
